Lanchonete/Lanchonete/ Features/Customization/Data/IngredientManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  IngredientInManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 1/13/21.
+//
+
+import Foundation
+
+public class IngredientManager {
+    var ingredients: [Ingredient]?
+    
+    init(with ingredients: [Ingredient]? = nil){
+        guard let ingredientsFinal = ingredients else {
+            return
+        }
+        self.ingredients = ingredientsFinal
+        return
+    }
+    
+    public func containsIngredient(ingredient: Ingredient) -> Bool {
+        if let ingredientsFinal = self.ingredients {
+            return ingredientsFinal.contains(ingredient)
+        }
+        return false
+    }
+}
```

Comment:
Is there a use case for the `ingredients` field being optional? If not, it should not be optional, simplifying the rest of the code. You can also set it to an empty list if that's a case you want to handle and will still simplify the rest of the class.

Reply:
You're totally right, not sure what I was smoking when I wrote this class 🤔  🌿 🚬

Reply:
Changed this